telegram/ext/basepersistence.py

```diff
@@ -128,12 +128,12 @@ def set_bot(self, bot: Bot) -> None:
         self.bot = bot
 
     @classmethod
-    def replace_bot(cls, obj: object) -> object:
+    def replace_bot(cls, obj: object) -> object:  # pylint: disable=R0911
```

Comment:
```suggestion
    def replace_bot(cls, obj: object) -> object:
```

Reply:
pylint complains about the methods having too many return statements, so moving that line to the `except Exception` will make the test fail …